### PR TITLE
Add reference to plugin dragdata

### PR DESCRIPTION
**Pull request by chrispahm**
Awesome Contribution Checklist:

- [x] I have read, and re-read the [Contributing Guidelines](https://github.com/chartjs/awesome/blob/master/CONTRIBUTING.md)
- [x] I have searched to ensure the suggested item doesn't exist on this list
- [x] This PR contains only one item

### Please Describe Your Addition

This PR adds a reference to [chartjs-plugin-dragdata](https://github.com/chrispahm/chartjs-plugin-dragdata), which allow users to drag data points on a chart.

plugin-dragdata demo

### CODE REVIEW

**Review by benmccann — DISMISSED** [review]
Looks cool!

File: README.md [hunk 1]
```diff
@@ -29,6 +29,7 @@ A curated list of awesome things related to [Chart.js](https://www.chartjs.org)
 - [crosshair](https://github.com/abelheinsbroek/chartjs-plugin-crosshair) - Adds a data crosshair to line and scatter charts.
 - [datalabels](https://github.com/chartjs/chartjs-plugin-datalabels) - Displays labels on data for any type of charts.
 - [deferred](https://github.com/chartjs/chartjs-plugin-deferred) -  Defers initial chart update until chart scrolls into viewport.
+- [dragData](https://github.com/chrispahm/chartjs-plugin-dragData) - Lets users drag data points on the chart.
```
Comment (simonbrunel) [inline README.md]: The plugin name and package name [should be lowercase](https://www.chartjs.org/docs/latest/developers/plugins.html#plugin-id): 

```
[dragdata](https://github.com/chrispahm/chartjs-plugin-dragdata)
```

Can you also add an id to [your plugin](/chrispahm/chartjs-plugin-dragData/blob/master/src/index.js#L157) that reflects the package name: `id: 'dragdata'` (similar to [this](/chartjs/chartjs-plugin-datalabels/blob/master/src/plugin.js#L123))

**Review by simonbrunel — CHANGES_REQUESTED** [review]
Thanks @chrispahm for your submission. Can you please address the issues before we can merge your PR?

File: README.md (hunk 1 above)
Comment (chrispahm) [reply README.md]: Thanks @simonbrunel for having a look!
I updated the plugin's name to be all lowercase, and added the required plugin id (see https://github.com/chrispahm/chartjs-plugin-dragdata/pull/28).
Also, the plugin name in this PR was updated accordingly. I hope this addresses all the issues mentioned?

**Review by simonbrunel — APPROVED** [review]

**Comment by simonbrunel** [comment]
@chrispahm thank you

**Pull request merged**